go/state/state_db.go

```diff
@@ -522,6 +532,17 @@ func (s *stateDB) CreateAccount(addr common.Address) {
 		}
 	})
 
+	// Reset transient storage.
+	s.transientData.ForEach(func(slot slotId, value common.Value) {
```

Comment:
I think this cannot happen. Transient state must be fresh for each transaction and a transaction cannot: create an account, modify its transient state, and create the same account again

Reply:
this storage should be cleared at every transaction, which it seems is no happening at the moment.